templates/tables-template.qmd

```diff
@@ -0,0 +1,109 @@
+---
+
+---
+
+
+<!--` reference1 -->
+# reference
+
+intro
+
+<!--` reference1/ -->
+
+```{r, echo=FALSE}
+main_db <- DBI::dbConnect(RSQLite::SQLite(), "../data/tables/main.sqlite")
+quary <- paste("SELECT", "*", "FROM", "table_of_contents")
+val_table <- DBI::dbGetQuery(main_db, quary)
+
+content_string <-""
+table_of_content <-"<ul>"
+
+for (row_index in 1:nrow(val_table)) {
+  if(is.na(val_table[row_index, ]$partType)){
+    next()
+  }
+  
+  working_row <- val_table[row_index, ]
+  
+  content_string <- paste0(content_string, working_row$partLabel, " (", working_row$partID, ") </br>", working_row$partDesc, "</br></br>")
```

Comment:
What is this for?

Reply:
To create a bigger space after part Description as it was cluttered when it was only one line break.

Reply:
To increase the space after a description for clarity a single line break was too cluttered.